### PR TITLE
keeping submodules up to date

### PR DESCRIPTION
**Pull request by zeke**
@kevinsawicki taught me and @jlord this One Weird Trick™

### CODE REVIEW

File: docs/development/source-code-directory-structure.md [hunk 1]
```diff
@@ -62,3 +62,30 @@ Electron
...
+```sh
+git submodule update --init --recursive
+```
+
+If you find yourself running this command often, you can create an alias for it
+in your `~/.gitconfig` file:
+
+```
+[alias]
+	su = git submodule update --init --recursive
```
Comment (lee-dohm) [inline docs/development/source-code-directory-structure.md]: Should this have the `git` keyword in it? I think aliases of normal commands with parameters leave the `git` out.

Comment (paulcbetts) [inline docs/development/source-code-directory-structure.md]: :point_up:

**Comment by kevinsawicki** [comment]
For anyone looking for a command that upgrades all submodules to their latest versions I also use this `git supgrade` alias:

```
[alias]
	supgrade = submodule foreach 'git fetch --prune --all && git checkout FETCH_HEAD'
```

File: docs/development/source-code-directory-structure.md (hunk 1 above)
Comment (zeke) [inline docs/development/source-code-directory-structure.md]: ✅

**Comment by zcbenz** [comment]
👍

**Pull request merged**